authors/apps/authentication/views.py

```diff
@@ -28,17 +34,99 @@ class RegistrationAPIView(GenericAPIView):
     serializer_class = RegistrationSerializer
 
     def post(self, request):
+        """
```

Comment:
Great doc-string description, I have learned something from this and I will be taking it up as well in the next few implementations. Thank you.

Reply:
We learn and progress, human evolution.